test/fixtures/imports.yml

```diff
@@ -14,6 +14,18 @@ loaded_import:
     2024-01-02,Amazon stuff,Shopping,200
   created_at: <%= 2.days.ago %>
 
+loaded_import_with_missing_data:
+  account: checking
+  raw_csv_str: |
+    date,name,category,amount
+    2024-01-01,,Food,20
+    2024-01-02,Amazon stuff,,200
+  normalized_csv_str: |
+    date,name,category,amount
+    2024-01-01,,Food,20
+    2024-01-02,Amazon stuff,,200
+  created_at: <%= 2.days.ago %>
+
```

Comment:
Small nitpick, but I was generally keeping these fixtures as the "base cases" and then using that `import_test_helper.rb` (or directly in the test block if the data is not shared) to provide variation strings for tests.  This is mostly to stay in line with [the guide](https://edgeguides.rubyonrails.org/testing.html#what-are-fixtures-questionmark):

 > Fixtures are not designed to create every object that your tests need, and are best managed when only used for default data that can be applied to the common case.

We can then build the variation object directly in the test.  My _guess_ is that we'll be adding several more of these as we add complexity to this flow.

Reply:
Makes sense, fixed 😄